students/JerryH/Lesson03/slicing.py

```diff
@@ -0,0 +1,14 @@
+def exchange_first_last(seq):
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+def every_other(seq):
+    return seq[::2]
+
+def remove_first_last(seq):
+    return seq[4:-4:2]
+
+def reverse_elements(seq):
+    return seq[-1::-1]
+
+def mid_last_first(seq):
+    return seq[(len(seq)//2)+2:(len(seq)//2)+3] + seq[-3:-2] + seq[2:3]
```

Comment:
review the solution for `mid_last_first`

given below sequence
```
>>> seq = (1,2,3,4,5,6,7,8,9)
```

should get back:
```
>>> thirds(seq)
(4, 5, 6, 7, 8, 9, 1, 2, 3)
```
but above function is returning:
```
>>> mid_last_first(seq)
(7, 7, 3)
```